spring-boot/admin/src/main/resources/templates/admin/fragments/proxies/list.html

```diff
@@ -166,5 +174,6 @@ <h2 th:text="#{admin.proxies.list.title}">[Proxies Administration]</h2>
         </div>
         <div th:include="admin/fragments/scripts :: scripts" th:remove="tag">Javascript Libraries</div>
         <div th:include="admin/fragments/scripts :: initializeDataTables" th:remove="tag">Javascript Libraries</div>
+        <script th:src="@{/webjars/fortawesome__fontawesome-free/6.0.0-beta2/js/all.js}"></script>
```

Comment:
What is the purpose of this script? There seem to be strange effects caused by this, there are some strange "animated icons" instead of the usual arrows:
![image](https://user-images.githubusercontent.com/6480551/149974054-391fd37c-459f-40ee-837f-35d62d959647.png)

Reply:
The single purpose of the script and the Webjar, which contains this js file, is to provide the docker icon for the download button. I found this to be the easiest solution but I have not considered possible side-effects. Feel free to share alternative approaches. 

Other approaches popping into my mind would be to just import the required css code into the html or to just download the css code at all and remove the Webjar.

Reply:
I just removed the Webjar and use another icon now.